packages/airnode-protocol/scripts/verify-deployment.ts

```diff
@@ -5,63 +5,48 @@
 // of the local compilation.
 
 import { assert } from 'console';
-import * as path from 'path';
-import * as fs from 'fs';
-import * as ethers from 'ethers';
 import { logger } from '@api3/airnode-utilities';
-
-// From https://stackoverflow.com/a/20525865/14558682
-// Gets a list of files in the directory recursively
-function getFiles(dir: any, files_: any) {
-  files_ = files_ || [];
-  const files = fs.readdirSync(dir);
-  for (const i in files) {
-    const name = path.join(dir, files[i] as string);
-    if (fs.statSync(name).isDirectory()) {
-      getFiles(name, files_);
-    } else {
-      files_.push(name);
-    }
-  }
-  return files_;
-}
+const hre = require('hardhat');
 
 async function main() {
-  const networks = ['mainnet', 'ropsten', 'rinkeby', 'goerli', 'kovan'];
-  const contracts = ['AccessControlRegistry', 'AirnodeRrp', 'RequesterAuthorizerWithAirnode'];
-  const credentials = JSON.parse(fs.readFileSync('credentials.json', 'utf8'));
-  const artifactFileNames = getFiles(path.join('artifacts', 'contracts'), []);
+  const contractNames = ['AccessControlRegistry', 'RequesterAuthorizerWithAirnode', 'AirnodeRrp'];
```

Comment:
I don't expect that we will add any other deployments to @api3/airnode-protocol. RrpBeaconServer and everything else will be deployed on other repos. Also, I'm not sure that I like to depend on the existing files in case some go missing by accident. I'll look into an alternative.

Reply:
I did this, though it's kinda meh https://github.com/api3dao/airnode/pull/962/commits/f8393fef6f5d6a53d17a670d4fbaa679b9e74778

Reply:
At least they are in a single place 👍🏻